x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -15,15 +15,14 @@ import { AgentName } from '../typings/es_schemas/ui/fields/agent';
  */
 
 export const AGENT_NAMES: AgentName[] = [
-  'java',
```

Comment:
This has been alphabetized so the mapping output stays consistent with what already exists, and there were two "java"s here.